skopt/tests/test_common.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
+
+from sklearn.utils.testing import assert_almost_equal
+from sklearn.utils.testing import assert_array_less
+from sklearn.utils.testing import assert_array_equal
+from sklearn.utils.testing import assert_equal
+from sklearn.utils.testing import assert_raises
+
+from skopt.benchmarks import branin
+from skopt.dummy_opt import dummy_minimize
+from skopt.gp_opt import gp_minimize
+from skopt.tests.test_tree_opt import MINIMIZERS as TREE_MINIMIZERS
```

Comment:
How can I make this work in legacy python? Alternatively should there be `skopt.utils` with a `all_minimizers()`? Trying to find a nice way to not repeat `MINIMIZERS` from the tree tests.